### ISSUE TITLE
section main

### ISSUE THREAD

**Issue by cegerard** [peluche1275/KatHTML#1]
https://github.com/peluche1275/KatHTML/blob/453ad2d095204b1c829fcd53381cf61327f6cf04/index.html#L29

Pour toi les articles ne font pas partie du contenu principale ?

**Comment by peluche1275** [comment]
Oui, je me suis dis que c'est du contenu changeant et que cela affecterai le SEO. @cegerard
